data/json/items/comestibles/fruit_dishes.json

```diff
@@ -22,6 +22,60 @@
     "type": "COMESTIBLE",
     "id": "fruit_leather",
     "name": { "str": "fruit leather" },
+    "conditional_names": [
+      { "type": "COMPONENT_ID", "condition": "apple", "name": { "str_sp": "%s, apple" } },
+      { "type": "COMPONENT_ID", "condition": "apricots", "name": { "str_sp": "%s, apricots" } },
+      { "type": "COMPONENT_ID", "condition": "banana", "name": { "str_sp": "%s, banana" } },
+      { "type": "COMPONENT_ID", "condition": "blackberries", "name": { "str_sp": "%s, blackberries" } },
+      { "type": "COMPONENT_ID", "condition": "blueberries", "name": { "str_sp": "%s, blueberries" } },
+      { "type": "COMPONENT_ID", "condition": "cherries", "name": { "str_sp": "%s, cherries" } },
+      { "type": "COMPONENT_ID", "condition": "elderberries", "name": { "str_sp": "%s, elderberries" } },
+      { "type": "COMPONENT_ID", "condition": "grapefruit", "name": { "str_sp": "%s, grapefruit" } },
+      { "type": "COMPONENT_ID", "condition": "grapes", "name": { "str_sp": "%s, grape" } },
+      { "type": "COMPONENT_ID", "condition": "huckleberries", "name": { "str_sp": "%s, huckleberries" } },
+      { "type": "COMPONENT_ID", "condition": "irradiated_apple", "name": { "str_sp": "%s, irradiated apple" } },
```

Comment:
Do irradiated fruits rot at the same rate as other non-irradiated fruits once dehydrated?

Reply:
Yep, pretty sure it's a vanilla reaction
![image](https://user-images.githubusercontent.com/36739484/140526730-966a64b1-9e9b-493c-bd7d-9ead69624ea4.png)

Reply:
OK I would get rid of the word irradiated so irradiated apples would just be dehydrated apples and so would regular apples if they are indistinguishable other than name once they are dehydrated.

Reply:
deleting the entire line of the irradiated parts would automatically pull the name apples since all the irradiated items originated from normal fruit